modules/install/pages/upgrade.adoc

```diff
@@ -1,34 +1,47 @@
 = Upgrade
-:description: To upgrade a Couchbase-Server cluster means to revise upwards the version of Couchbase Server that is running on every node.
+:description: To upgrade a Couchbase-Server cluster means to upgrade the version of Couchbase Server that is running on every node.
+
+:erlang-upgrade-note: The upgrade to Erlang support in Couchbase Server 7.2.4 requires that Couchbase first be upgraded to version 7.1.0 or later, before upgrading to 7.2.4.
```

Comment:
The upgrade to Erlang support in Couchbase server 7.2.4 requires that you first upgrade Couchbase to version 7.1.0 or later before upgrading to version 7.2.4.